src/test/scala/fi/oph/koski/api/OppijaValidationTaiteenPerusopetusSpec.scala

```diff
@@ -145,6 +144,7 @@ class OppijaValidationTaiteenPerusopetusSpec
 
     "Opiskeluoikeuden lisäämisen oikeudet" - {
       "pääkäyttäjä voi luoda hankintakoulutuksena järjestettävän opiskeluoikeuden" in {
+        resetFixtures()
```

Comment:
Pohdiskelen, että olisiko loogisempaa laittaa tämä resetFixtures niiden testien loppuun, jotka muuttavat tietokantaa. Tässä tiedostossahan on paljon testejä, joissa tietokanta ei muutu, kun testi tyssää käyttöoikeuksiin.

En kyllä osaa sanoa, kumpi on parempi. Loppuun laittamalla pystyisi estämään jotain turhia resetointeja, mutta voi muuten olla epäselvempää.

Reply:
Joo, osa näistä fikstuurien resetoinneista on varmasti tarpeettomia ja nehän myös hidastaa testien ajoa. Voisin katsoa niiden käyttöä vielä. Lisäsin näitä toisaalta myös ihan selkeyden vuoksi, kun alkoi itsellä tulla epävarma olo että mikä tietokannan tila on kunkin testin kohdalla.

Reply:
Yritin ainakin vähentää resetFixturen käyttöä, mutta sitä on kyllä vielä runsaasti viljeltynä. Pystyisi varmaan vieläkin vähentämään mutta mielestäni testien luettavuus kärsii ja voi helposti tulla ajatusvirheitä, esim yritetään mitätöidä opiskeluoikeus joka on jo mitätöity aiemmassa testissä tai luoda opiskeluoikeus joka on jo luotu aiemmassa testissä.